proteus.go

```diff
@@ -30,6 +30,8 @@ func transformToProtobuf(packages []string, generate generator) error {
 	r.Resolve(pkgs)
 
 	t := protobuf.NewTransformer()
+	t.SetStructSet(createStructTypeSet(pkgs))
```

Comment:
If `Transformer` needs to hold state, I prefer passing the packages directly to the constructor and then set everything up instead of this.